hasura/metadata/databases/default/tables/public_action.yaml

```diff
@@ -50,6 +50,8 @@ insert_permissions:
         - app_flow_on_complete
         - webhook_pem
         - webhook_uri
+        - return_to_ios
```

Comment:
Given we want these to be deep or universal links is there a way to rename these better?